BAZEL.md

```diff
@@ -300,6 +300,14 @@ Bazel command (`run`, `test`, etc.). If applicable, you can also define Bazel
 command-line flags or command-line flags to the executable. Click on "Apply",
 or "OK" to add the run configuration.
 
+#### Attaching sources to scala library
+
+If you do not have the scala library sources linked (you only see the decompiled
+ sources only), you can attach it manually by selecting the `Choose sources...` 
+ button on the yellow bar at the top, and selecting `scala-library...-src.jar`. 
+(you might need to update dependencies beforehand by running 
+`bazel run @unpinned_maven//:pin`)
```

Comment:
You shouldn't need to do this unless you change _bazel-java-deps.bzl_. What does it achieve?

I would have thought that asking the user to _Sync Project with BUILD Files_ would be enough.

Reply:
Sounds great! removing it